### ISSUE TITLE
Process Instance Modification shows special behaviour

### ISSUE THREAD

**Issue by jonathanlukas** [camunda/camunda-platform-docs#1727]
Process Instance Modification shows a special behaviour when activating a joining parallel gateway.

Expected: The activated joining gateway starts the joining behaviour on activation.

Actual: The activated joining gateway continues on activation, even if this would require more than 1 token.

I discussed this with @saig0 [here](https://camunda.slack.com/archives/CSQ2E3BT4/p1677095270163419).

We agreed that documenting this would be nice.

**Comment by korthout** [comment]
Marking this as upcoming as it is low-hanging fruit and directly impacts users.

**Comment by megglos** [comment]
@korthout is this still valid or did something change in that regard ?

**Comment by korthout** [comment]
Yes this is still valid. The joining gateway is activated without applying the joining behavior. A single token is enough to continue the process from the joining parallel gateway.

PS: Not sure how I missed your question, but here's an answer finally
